en/snap/with-plugins.md

```diff
@@ -56,269 +57,309 @@ Here is the list of Content Management System plugins and extensions that are su
+* WordPress v3.9 or later **|** Tested up to v5.x
+* WooCommerce v2 or later **|** Tested up to v3.5.2
+* PHP version v5.4 or later
+* MySQL version v5.0 or later
 * PHP CURL enabled server/host
 
-#### Installation:
+#### Installing Midtrans WooCommerce Plugin
+Select **any one** of the installation options given below.
 
-Choose **one** from installation options **A or B**:
+#### **Simple Installation**
 
-#### A. Simple Installation
-   1. Login to your Wordpress admin panel.
-   2. Go to Plugins menu, click add new. Search for Midtrans-WooCommerce plugin.
-   3. Install and follow on screen instructions.
-   4. Proceed to Configuration Process.
+To install Midtrans-WooCommerce plugin, follow the steps given below.
+  1. Login to your WordPress administration panel.
+  2. Go to **Plugins** menu.
+  3. Click **add new**.
+  4. Search for **Midtrans-WooCommerce plugin**.
+  5. Click **Install Now** and follow on-screen instructions.
+  6. Proceed to [Configuring Midtrans WooCommerce Plugin](#configuring-midtrans-woocommerce-plugin).
 
-If you are unable to install using this method, please proceed with [manual installation below](#b-manual-installation).
+If you are unable to install, proceed to [Manual Installation](#manual-installation).
 
-#### B. Manual Installation
+#### **Manual Installation**
+
+To install Midtrans-WooCommerce plugin manually, follow the steps given below.
    1. Download the plugin from [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip).
    2. Extract the plugin, then rename the modules folder as **midtrans-woocommerce**.
    3. Upload the unzipped plugin folder to your WordPress installation's `./wp-content/plugins/` directory.
-   4. Install and activate the plugin from plugins menu within the WordPress admin panel
+   4. **Install and activate** the plugin from plugins menu within the WordPress administration panel.
    5. Proceed to Configuration Process.
-   
-#### WooCommerce Plugin Configuration Process
-1. Go to **WooCommerce -> Settings -> Payments -> Midtrans** menu, fill the configuration fields.
-    - Fill **Title** with text button that you want to display to customer
-    - Select **Environment**. `Sandbox` for testing transaction and `Production` for real transaction
-    - Fill **Merchant ID, Client Key, and Server key**. You can find [this credential on Midtrans MAP Dashboard](/en/snap/preparation.md).
-    - Other configuration are optional, you may leave it as is.
+
+#### Configuring Midtrans WooCommerce Plugin
+To configure WooCommerce plugin, follow the steps given below.
+1. Go to **WooCommerce > Settings > Payments > Midtrans** menu.
+2. Enter **Title**. This text appears on the button displayed to the customer.
+3. Select **Environment** from dropdown. `Sandbox` for testing transaction and `Production` for real transaction.
+4. Enter **Merchant ID**.
+5. Enter **Client Key**.
+6. Enter **Server key**.
+   For more details, refer to [Preparation](/en/snap/preparation.md).
```

Comment:
Why did you breakdown the sub-step of 1 into its own numbered lists? it make the step numbers unnecessary look longer. Also it is a group of configurations step within a menu, it make sense to leave it as sub-step. 

Happen to other parts of this page too.

Reply:
As a style, every small step that the user has to perform is required to be listed into separate steps to avoid any confusion. If you think this is not helping, I can change it back to how it was before.

Reply:
Yes, let's revert the change of sub-steps converted to numbered steps. Because numbered steps does make the steps seems longer.

Reply:
Sure. I will do them and commit the changes today.

Reply:
I have made the changes in other sections. Since in the original document, this heading has only one 1 item in the list, I have kept the numbered list.